tool/logs/testdata/compactions

```diff
@@ -120,7 +120,7 @@ reset
 
```

Comment:
I will plan to update the logs in this test file if y'all are cool with my log line updates.